wsinfer/write_geojson.py

```diff
@@ -83,8 +83,15 @@ def parallelize_geojson(csvs: list, results_dir: Path, num_workers: int) -> None
     if not results_dir.exists():
         raise FileExistsError(f"results_dir does not exist: {results_dir}")
     if output.exists():
-        # raise FileExistsError("Output directory already exists.")
-        shutil.rmtree(f"{output}")
+        geojsons = list((results_dir / "model-outputs-geojson").glob("*.json"))
+        
+        # Makes a list of filenames for both geojsons and csvs
+        geojson_filenames = [filename.name.split(".")[0] for filename in geojsons]
+        csv_filenames = [filename.name.split(".")[0] for filename in csvs]
+
+        # Makes a list of new csvs that need to be converted to geojson
+        csvs_new = [filename for filename in csv_filenames if filename not in geojson_filenames]
+        csvs_final = [path for path in csvs if path.name.split(".")[0] in csvs_new]
```

Comment:
This can be simplified. Make use of the `pathlib.Path` object. For example, instead of `filename.split(".")[0]`, you can use `filename.stem`. Also to get the CSVs that need to be converted, you can use something like this (untested):

```python
[csv for csv, geojson in zip(csvs, geojsons) if not geojson.exists()]
```

Reply:
@kaczmarj I'm not sure whether this will add the corresponding files. Let's say I have 3 csv files (CMU-1.csv, CMU-2.csv and CMU-3.csv) and I have only 2 geojsons (CMU-1.json and CMU-3.json). This piece might not generate the second json.

Reply:
i'm not sure either. could you test it?

Reply:
Yes, it isn't working

Reply:
oh i think i understand why. because `geojsons` and `csvs` will not be the same length... let's use your implementation that you have currently, but replace the `filename.name.split(".")[0]` with `path.stem`

```python
[p.stem for p in geojsons]
```

Reply:
Yes precisely. Will do

Reply:
thanks for catching that problem